README.md

```diff
@@ -1,41 +1,50 @@
 [![Build Status](https://dev.azure.com/responsibleai/fairlearn/_apis/build/status/Nightly?branchName=master)](https://dev.azure.com/responsibleai/fairlearn/_build/latest?definitionId=23&branchName=master) ![MIT license](https://img.shields.io/badge/License-MIT-blue.svg) ![PyPI](https://img.shields.io/pypi/v/fairlearn?color=blue)
 
-# fairlearn
+# Fairlearn
+
+The Fairlearn project seeks to enable anyone involved in the development of artificial intelligence (AI) systems to assess their system's fairness and mitigate the observed unfairness. The Fairlearn repository contains a Python package and Jupyter notebooks with the examples of usage.
+
+ #### Fairlearn components ####
+ * Assessment dashboard: high-level and detailed views, for assessing which groups are negatively impacted. 
+ * Mitigation strategies: easy to incorporate into existing machine-learning pipelines.
+
+Together, these components empower data scientists and business leaders to navigate any tradeoffs between fairness and performance, and to select the mitigation strategy that best fits their needs.
```

Comment:
Rather than 'performance' should that be 'accuracy'?

Reply:
Agreed that the terminology should be "fairness" and "accuracy" (at least for the time being). This has been edited now.